script/interface.lua

```diff
@@ -105,7 +105,9 @@ Raised every UpdateInterval, after delivery generation
     force : LuaForce,
     train : LuaTrain,
     from : string,
+    from_id : int,
     to : string,
+    to_id : integer,

```

Comment:
Hey, sorry to bother, but is it `int` or `integer`? 😆 `from_id` is `int`, `to_id` is `integer`. A bit inconsistent.

Reply:
Inconsistencies in type description where already fixed this morning.